arbitrum-docs/node-running/how-tos/running-a-full-node.mdx

```diff
@@ -76,10 +69,14 @@ content-type: how-to
   chmod -fR 777 /data/arbitrum
   ```
 
+### Watchtower mode
+- By default, the full node will work in Watchtower mode. This means that the node watches the on-chain assertions and if it disagrees with them, it will log an error containing the string `found incorrect assertion in watchtower mode`.
+- You can deactivate this mode by using the parameter `--node.staker.enable=false`.
```

Comment:
Watchtower mode adds a small amount of execution and memory overhead.  You can deactivate...